confdb/aclocal_modules.m4

```diff
@@ -41,7 +41,7 @@ dnl internal routine, $1 is the extra cflags, hwloc_embedded_dir is m4 macro
 dnl defined to be the path to embedded hwloc.
 AC_DEFUN([PAC_CONFIG_HWLOC_EMBEDDED],[
     PAC_PUSH_FLAG([CFLAGS])
-    CFLAGS="$1"
+    CFLAGS="$USER_CFLAGS $1"
```

Comment:
I believe this will fix the reported issue, but it looks like it might not work for standalone Hydra builds? For whatever reason, `PAC_RESET_ALL_FLAGS` only sets `USER_CFLAGS` and others if `"$FROM_MPICH" = "yes"`.

Reply:
Which `PAC_RESET_ALL_FLAGS` are you referring to?

When `FROM_MPICH` is not set (building standalone hydra), it will call `PAC_CONFIG_HWLOC_EMBEDDED()`, i.e. with `$1` empty, no visibility flag. It will still reset `CFLAGS="$USER_CFLAGS  ` -- just with extra spaces.

Reply:
My mistake, `PAC_RESET_ALL_FLAGS` doesn't set `USER_*FLAGS`. What I meant was `PAC_PREFIX_ALL_FLAGS(USER)` which saves the initial `CFLAGS` into `USER_CFLAGS`. The Hydra configure doesn't do this, so as far as I can tell, `USER_CFLAGS` will always be empty in standalone Hydra.

Reply:
I see. We need do the same for hydra then. I'll patch it up.